# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.cpp
# Commit: 98848e63dfbf
# Full Hash: 98848e63dfbf5702726d3e07fb92855263312fed
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r b668d0c685e8 -r 98848e63dfbf netwerk/cookie/CookieService.cpp
--- a/netwerk/cookie/CookieService.cpp	Wed Jan 08 19:25:59 2025 +0000
+++ b/netwerk/cookie/CookieService.cpp	Wed Jan 08 19:26:00 2025 +0000
@@ -1808,5 +1808,14 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP
+CookieService::TestGet3PCBExceptions(nsTArray<nsCString>& aExceptions) {
+  aExceptions.Clear();
+
+  mThirdPartyCookieBlockingExceptions.GetExceptions(aExceptions);
+
+  return NS_OK;
+}
+
 }  // namespace net
 }  // namespace mozilla